ml-agents/mlagents/trainers/learn.py

```diff
@@ -191,33 +192,15 @@ def write_timing_tree(output_dir: str) -> None:
         )
 
 
-def maybe_add_samplers(
-    sampler_config: Optional[Dict], env: SubprocessEnvManager, run_seed: int
-) -> None:
-    """
-    Adds samplers to env if sampler config provided and sets seed if not configured.
-    :param sampler_config: validated dict of sampler configs. None if not included.
-    :param env: env manager to pass samplers via reset
-    :param run_seed: Random seed used for training.
-    """
-    if sampler_config is not None:
-        # If the seed is not specified in yaml, this will grab the run seed
-        for offset, v in enumerate(sampler_config.values()):
-            if v.seed == -1:
-                v.seed = run_seed + offset
-        env.set_env_parameters(config=sampler_config)
-
-
-def try_create_meta_curriculum(
-    curriculum_config: Optional[Dict], env: SubprocessEnvManager, restore: bool = False
-) -> Optional[MetaCurriculum]:
-    if curriculum_config is None or len(curriculum_config) <= 0:
+def try_create_param_manager(
+    config: Optional[Dict[str, EnvironmentParameterSettings]],
+    run_seed: int,
+    restore: bool = False,
+) -> Optional[EnvironmentParameterManager]:
+    if config is None:
```

Comment:
Can we return an empty EnvironmentParameterManager in this case? I think it would simplify some of the None checks in TrainerController.